src/api/modules/user_tokens/index.ts

```diff
@@ -0,0 +1,11 @@
+import type { PermissionType } from "~applications/permissions";
+import type { ModuleProperties } from "~api/module";
+
+const permissions: PermissionType[] = ["ACCESS_BALANCES"];
```

Comment:
Should this permission still be called `ACCESS_BALANCES` when the api is renamed ?

Reply:
@7i7o Thoughts?

Reply:
I agree with @pawanpaudel93 that `ACCESS_TOKENS` is a better name for the added permission